extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -45,8 +45,9 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 			maxDepth = undefined;
 		}
 		let workspacePaths: string[] = workspaceFolders.map(a => a.uri.fsPath);
-		for (let path of workspacePaths) {
-			let tableOfContentPaths = await glob([path + '/**/_data/toc.yml'], { deep: maxDepth });
+		for (let workspacePath of workspacePaths) {
+			let p = path.join(workspacePath, '**', '_data', 'toc.yml').replace(/\\/g, '/');
```

Comment:
why are you doing a replace of `\\` with `/`? the path library handles what the format should be for the current platform.

Reply:
for some reason fast-glob only works with forward slashes. It wasn't finding the toc.yml files when I passed in the path with backslashes

Reply:
if you replace it for windows, does it still work on mac?

Reply:
yes I tested on my mac